plugin-filters/taxonomy-filters.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Provides various filters for
+ * taxonomies
+ */
+namespace FinAid\Utils\Plugins;
+
+function disable_taxonomy_archives() {
+	if ( is_category() || is_tag() || is_date() || is_author() ) {
```

Comment:
`is_date()` and `is_author()` views are already disabled in the UCF WP Theme here: https://github.com/UCF/UCF-WordPress-Theme/blob/master/includes/config.php#L372